client/src/app/components/contact/contact.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, OnInit } from '@angular/core';
+import { FormGroup, FormBuilder, FormControl, Validators } from '@angular/forms';
+
+@Component({
+  selector: 'op-contact',
+  templateUrl: './contact.component.html',
+  styleUrls: ['./contact.component.scss']
+})
+export class ContactComponent implements OnInit {
+  contactForm: FormGroup;
+  nameControl;
+  emailControl;
+  contentControl;
+  displayErrs: boolean = false;
+
+
+  constructor(private formBuilder: FormBuilder) {
+      this.buildForm();
+  }
+
+  checkAll(){
+      if(!this.contactForm.valid) {
+          this.displayErrs = true;
+      }
+  }
+  buildForm() {
+      this.contactForm = this.formBuilder.group({
+          name: this.formBuilder.control(null, [Validators.minLength(2),
+                                                Validators.required]),
```

Comment:
Just FYI: We're working with GitHub style, so each line can be 120 characters (On Django too)